recipe/conda_build_config.yaml

```diff
@@ -32,18 +32,20 @@ gsl:
   - '2.7'
 libaprutil:
   - '1'
+libcxx:
+  - '16'
```

Comment:
Wait, I think this should just be osx.

Reply:
It's `- libcxx {{ libcxx }}.*  # [not aarch64]` over in `meta.yaml`

Reply:
Is it OK, wasteful, or harmful as is?  I've been building this way on Linux for a while.